src/main.rs

```diff
@@ -136,7 +152,7 @@ fn main() {
 
     // Parse used-memory file location or use default
     let used_memory_file = matches.opt_str("used-memory-file").
-        unwrap_or_else(|| "/sys/fs/cgroup/memory/memory.usage_in_bytes".to_string());
+        unwrap_or_else(|| "".to_string());
```

Comment:
putting the "find the default file" logic here would be nice ... is there a reason not to ?